playbooks/roles/ocp-operator-api/README.md

```diff
@@ -0,0 +1,59 @@
+ocp-cluster-observability-operator
+=========
+
+This role is used to validate operator development using operator-sdk.
+
+Requirements
+------------
+
+- OCP 4.x healthy cluster on PowerVS.
+
+
+Role Variables
+--------------
+
+| Variable                                   | Required | Default                                                                              | Comments                                                                                                                       |
+|--------------------------------------------|----------|--------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| op_api_role_enable                            | no       | false                                                                                | Set it to true to run this playbook                                                                                            |
+| op_cleanup                            | no       | false                                                                                | Set it to true to do operator cleanup.                                                                                            |
+| build_operator_image                   | no       | true                                                                                 | Set it to true to build the operator image                                                                      |
```

Comment:
Since the default is true.
```suggestion
| build_operator_image                   | no       | true                                                                                 | Set it to false to not build the operator image                                                                      |
```